integration_tests/mockApis/educationAndWorkPlanApi.ts

```diff
@@ -978,6 +978,98 @@ const stubGetEducation404Error = (prisonNumber = 'G6115VJ'): SuperAgentRequest =
     },
   })
 
+const stubGetActionPlanReviews = (prisonNumber = 'G6115VJ'): SuperAgentRequest =>
```

Comment:
I've added the basic wiremock stubs to get the action plan reviews - happy path, and 2 sad paths (404 and 500) for as and when we need them.